src/reflect/scala/reflect/runtime/JavaUniverseForce.scala

```diff
@@ -123,6 +123,7 @@ trait JavaUniverseForce { self: runtime.JavaUniverse  =>
     this.termNames
     this.nme
     this.sn
+    this.undetBaseTypeSeq
```

Comment:
This was triggered by making `undetBaseTypeSeq` a lazy val. Explanation of why is above.